src/renderer/pages/Operations/components/LogModal.tsx

```diff
@@ -35,11 +35,32 @@ const EventMessage: Partial<Record<SigningStatus | 'INITIATED', string>> = {
   ERROR_CANCELLED: 'log.errorCancelledMessage',
 } as const;
 
+type WalletsMap = Map<Wallet['id'], Wallet>;
+
+const getFilteredWalletsMap = (wallets: Wallet[]): WalletsMap =>
```

Comment:
Why not this one?
Arrow function without explicit `return` is ok for one-liners but here is another story

```javascript
type WalletsMap = Record<Wallet['id'], Wallet>;

const getFilteredWalletsMap = (wallets: Wallet[]): WalletsMap => {
  return wallets.reduce<WalletsMap>((acc, wallet) => {
    // 2nd condition for legacy multisig
    if (walletUtils.isValidSignatory(wallet) || walletUtils.isMultiShard(wallet)) {
      acc[wallet.id] = wallet;
    }

    return acc;
  }, {});
};
```